examples/sensors.rs

```diff
@@ -14,11 +14,15 @@ async fn main() -> TockResult<()> {
     let mut temperature = TemperatureSensor;
     let mut light = AmbientLightSensor;
     let mut ninedof = unsafe { Ninedof::new() };
+    let context = timer::DriverContext::create()?;
+    let mut driver = context.create_timer_driver().unwrap();
```

Comment:
Any reason to `unwrap` here, where the rest of the function uses returns an error with `?` instead?

Reply:
solved.